contributor-guide/modules/ROOT/pages/testing/test-policy.adoc

```diff
@@ -0,0 +1,53 @@
+= Test Policy
+:navtitle: Test Policy
```

Comment:
Is this term commonly used? I've never heard it before.

Reply:
@alandefreitas The term is used in the legacy intro. I could change it to "Test Procedures", or "Test Overview" or similar?

Reply:
>  The term is used in the legacy intro

Oh OK.

> I could change it to "Test Procedures", or "Test Overview" or similar?

It's up to you. "Test policy" might be good since it was already in the legacy intro. We can always change if someone complains.